### DISCUSSION TITLE
Syncing with Syncthing?

### DISCUSSION THREAD

**Discussion by szemlicka** [advplyr/audiobookshelf#315] [Ideas]
Hi,
I want to discuss this if this makes sense. I like the capability of Syncthing. At the moment I sync all my audiobooks with this software. I just copy the audiofiles I want at my mobile to a special folder at my NAS, and Syncthing makes the rest and push that data to my mobile.

Make it sense to add a "homedirectory" to each user and s sync button to the album overview? The sync button just copy the files (including file structure) the the home dir and from that point Syncthing makes the rest?

I don't know if this makes sense or not.

Any comment?

**Comment by szemlicka** [comment]
I find it a pity that this idea does not find friends.

**Reply by advplyr** [reply]
I think I know what you mean, but I'm not sure many other people have this use-case.
Also, as the mobile app improves the goal is to have downloading audiobooks from your server to your phone be a smooth and simple process.

**Comment by szemlicka** [comment]
I love Syncthing. It automaticly sync if wifi is connected.  And in additional I can use my own prefered player (Smart Audio Player). 
Ok, the setup is not so simple. But for my family its perfect. With the function of a "virtual" home dir in audiobookshelf, you only need one click (to copy files to a local folder) and from that point syncthing is syncing. And as for the implementation, the time needed to "implement" the home dir and a copy button is small, but the profit is enormous.

At the moment I mad it manually but my wife love it. With a little magic new audiobook files are come over to her phone without any touch. And the best, with this I can save and sync the status of the played books automaticly. The preferred player save it in a file which I also sync...:-)  

But I also understand that you want to build that function into the mobile phone app.

**Reply by Penguinjumper** [reply]
> And the best, with this I can save and sync the status of the played books automaticly. The preferred player save it in a file which I also sync...:-)
> 
> But I also understand that you want to build that function into the mobile phone app.

Have you ever found a way to do this? I've been looking everywhere!
